lib/vscode/coder.js

```diff
@@ -5,7 +5,7 @@ const _ = require("underscore");
 const buildfile = require("./src/buildfile");
 const common = require("./build/lib/optimize");
 const util = require("./build/lib/util");
-const deps = require("./build/dependencies");
+const deps = require("./build/lib/dependencies");
```

Comment:
I must have accidentally imported this. It doesn't look like it's used anywhere.

Reply:
Ah, I didn't even think to check! Guess we can remove then